pkg/internal/export/otel/traces.go

```diff
@@ -347,6 +355,11 @@ func traceName(span *request.Span) string {
 		return span.Path
 	case request.EventTypeHTTPClient:
 		return span.Method
+	case request.EventTypeSQLClient:
+		// We don't have db.name, but follow "<db.operation> <db.name>.<db.sql.table_name>"
+		operation := span.Method
+		table := span.Path
+		return operation + " ." + table
```

Comment:
Excellent.  Double checking the otel semantics: "If db.sql.table is not available due to its semantics, the span SHOULD be named <db.operation> <db.name>"
I think omitting the empty space (since we don't have db.name) makes sense.
Thanks!